doc/source/cluster/configure-manage-dashboard.md

```diff
@@ -224,6 +225,10 @@ By default, Ray Dashboard assumes Prometheus is hosted at `localhost:9090`. You
 
 Then, configure `RAY_PROMETHEUS_HOST` environment variable properly as stated above. For example, if Prometheus is hosted at port 9000 on a node with ip 55.66.77.88, set `RAY_PROMETHEUS_HOST=http://55.66.77.88:9000`.
 
+If the Prometheus requires additional headers for authentication, set `RAY_PROMETHEUS_HEADERS` in one of the following JSON formats for Ray and Grafana to carry them to Prometheus:
```

Comment:
what's the relationship between `RAY_PROMETHEUS_HEADERS` and Grafana?

```suggestion
If the Prometheus requires additional headers for authentication, set `RAY_PROMETHEUS_HEADERS` in one of the following JSON formats for Ray and Grafana to carry them to Prometheus:
```

Reply:
`RAY_PROMETHEUS_HEADERS` will be included in the `grafana/provisioning/datasources/default.yml` which is generated by Ray for Grafana to know information about the Prometheus, such as its address, and request headers to carry on. But to avoid confusion, I have changed the term `Ray and Grafana` to `Ray dashboard`.